src/main.mjs

```diff
@@ -1,16 +1,30 @@
-// This is the example "Hello World" from:
-// https://expressjs.com/en/starter/hello-world.html
-const express = require('express');
-const morgan = require('morgan');
+//****************************************************************
+// INFO:
+// This file contains the main logic and routing for the server
+//****************************************************************
+
+//****************************************************************
+// Import and exports (ES6 style)
+//****************************************************************
```

Comment:
Project wide: Maybe lets not use this commenting style. Its excessively verbose for what ends up being often one line comments.